activerecord/CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+*   Add option to index errors in nested attributes
+
+    For models which have nested attributes, errors within those models will
+    now be indexed if :index_errors is specified when defining the nested
+    attributes, or if its set in the global config.
+
+    *Michael Probber and Terence Sun*
```

Comment:
You should add the names to the commit message too.

http://contributors.rubyonrails.org/faq#credit-several-people